packages/hash/frontend/src/pages/[account-slug]/[page-slug].page.tsx

```diff
@@ -362,7 +464,11 @@ const Page: NextPageWithLayout<PageProps> = ({ blocks }) => {
 
         <CollabPositionProvider value={[]}>
           <PageBlock
-            accountId={routeAccountId}
+            accountId={
+              pageWorkspace.kind === "user"
+                ? pageWorkspace.userAccountId
+                : pageWorkspace.orgAccountId
```

Comment:
as above, this could be simplified with a rename

Reply:
Addressed in https://github.com/hashintel/hash/pull/1650/commits/936a4734986f4d4cdfdc74feb321c997728a2e8c